gdx/src/com/badlogic/gdx/utils/Json.java

```diff
@@ -817,6 +826,16 @@ public void readFields (Object object, JsonValue jsonMap) {
 		}
 	}
 
+	/** Override to allow subclasses to use fake field names for other purposes in {@link #readFields(Object, JsonValue)} without
+	 * triggering an exception for unknown fields.
+	 * @param type The object type being read.
+	 * @param fieldName A field name encountered in the Json map for which there is no actual field.
+	 * @return Whether the field name should be treated as known, i.e., not throw an exception when encountered in
+	 *         {@code readFields()}. */
+	protected boolean isNonexistantFieldKnown (Class type, String fieldName) {
```

Comment:
I'm not a fan of this function name. How about `ignoreUnknownField`? Javadoc can explain it is only called when a JSON name doesn't have a matching field and `ignoreUnknownFields` is false.